openlibrary/plugins/upstream/account.py

```diff
@@ -402,6 +403,45 @@ def GET(self):
         # Show failed message without thinking.
         return render['account/verify/failed']()
 
+class real_time_email_verification(delegate.page):
+    path = "/account/email/rtverify"
+    
+    def GET(self):
+        i = web.input(email='')
+        result = {
+                'output': False,
+                'message': 'Must be a valid email address'
+            }
+        if(bool(re.match('.*@.*', i.email))):
+            ol_account = OpenLibraryAccount.get(email=i.email)
+            if(not(ol_account)):
+                result.output = True
+                result.message = "Email is available"
+            else:
+                result.message = "Email already registered"
+        return delegate.RawText(simplejson.dumps(result),
+                                content_type="application/json")
+
+class real_time_screenName_verification(delegate.page):
+    path = "/account/sceenname/rtverify"
+    
+    def GET(self):
+        i = web.input(name='')
+        username = i.name
+        result = {
+                'output': False,
+                'message': 'Letters and numbers only please, and at least 3 characters.'
+            }
+        if(username.isalnum() and len(username)>=3):
```

Comment:
```suggestion
        if username.isalnum() and len(username)>=3:
```

Reply:
Done

Reply:
Not done.

Reply:
Ohh thought I included this in the last pr. Now done, thanks :)